ReactiveUI.Fody.Tests/Issues/Issue41Tests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using NUnit.Framework;
+using ReactiveUI.Fody.Helpers;
+
+namespace ReactiveUI.Fody.Tests.Issues
+{
+    [TestFixture]
+    public class Issue41Tests
+    {
+        [Test]
+        public void PropertyChangeNotificationRaisedForDerivedPropertyOnIntPropertySet()
+        {
+            // Arrange
+            var model = new TestModel();
+            var expectedInpcPropertyName = "DerivedProperty";
```

Comment:
nit: change all these strings to use the `nameof` operator:  i.e. `nameof(TestModel.DerivedProperty)`